src/test/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/BestemmendeFravaersdagKtTest.kt

```diff
@@ -35,14 +34,12 @@ class BestemmendeFravaersdagKtTest : FunSpec({
                 ),
             )
 
-            actual shouldBe expected
+            bf shouldBe 1.januar
```

Comment:
Ingen biggie, men jeg foretrekker at expected settes én gang, slik det var tidligere - og deretter gjenbruke variabelen i videre oppsett og verifisering.
Det gjør testene mer robuste (slipper å endre flere steder, om man har behov for å endre på testdata)

Reply:
Ikke helt sikker på om jeg skjønte hva du mente. Tenker du på denne endringen eller en annen?

Reply:
Denne endringen - tidligere settes expected en gang, så kan den gjenbrukes / refereres til gjennom hele testen. 
Nå settes det ikke fast (1.januar), men nevnes (bare implisitt) i oppsettet - 1. januar er bare en dato, som alle de andre, det kommer ikke tydelig fram at akkurat den datoen / det feltet er viktig. 

Og nå må den gjentaes - 1. januar er en "kopi" - ny instansiering - i verifiseringssteget

Reply:
Ok, da er jeg med. Jeg skal reversere den endringen.
Jeg er stort sett enig at det er fint å gjøre det på den måten, og gjør det som oftest. Et unntak her er at jeg ikke bruker `expected` som argument i funksjonen som testes fordi jeg synes at det blir vanskeligere å lese. Det er spesifikt for periodene, hvor initialiseringen er naturlig lesbar.